src/main/resources/templates/register.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <link href="bootstrap/css/bootstrap.min.css"
```

Comment:
We should probably look to create a generic head, that is imported both into the Login, Registration and Template pages. Silly to keep linking bootstrap multiple times